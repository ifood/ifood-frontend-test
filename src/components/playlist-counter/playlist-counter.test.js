import React from 'react';
import PlaylistCounter from './playlist-counter';
import { shallow } from 'enzyme';

// Mounting - Full DOM rendering including child components
// Shallow - Renders only the single component, not including its children,
// this is useful to isolate the component for pure unit testing.
// Render - Renders to static HTML, including children

it('should render correctly with props', () => {
    const playlists = ['one', 'two'];
    const component = shallow(<PlaylistCounter playlists={playlists}/>);
    expect(component).toMatchSnapshot();
});

