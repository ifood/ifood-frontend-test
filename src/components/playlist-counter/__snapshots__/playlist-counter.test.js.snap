// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with props 1`] = `
<div
  style={
    Object {
      "color": "#000",
      "display": "inline-block",
      "fontFamily": "Papyrus",
      "fontSize": "20px",
      "lineHeight": "30px",
      "marginBottom": "20px",
      "width": "40%",
    }
  }
>
  <h2>
    Number of playlists: 
    <span
      style={
        Object {
          "color": "blue",
          "fontWeight": "bold",
        }
      }
    >
      2
    </span>
  </h2>
</div>
`;
